aiqleads/tests/core/test_project_tracking.py

```
@@ -3,16 +3,21 @@
 import os
 import json
 import pytest
+import shutil
 from datetime import datetime
 from typing import Dict
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)